### PR TITLE
[W44] Ray Lin: 11/01 - 11/02

### PR DESCRIPTION
**Pull request by raylin**

### CODE REVIEW

File: RayLin.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
-# 10/24 ~ 10/28
+# 11/01 ~ 11/02
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
-    - r+dholbert
+    - landing
```
Comment (timdream) [inline RayLin.md]: I don't see a new `checked-needed`. What's the hold up?

File: RayLin.md [hunk 2]
```diff
@@ -19,6 +18,9 @@
 		- the appearance of small size video - Bug 1311700
 		- should show error state and error text should fit on most video size
 		- top level synthetic media - buttons such as play, pause, mute, CC, fullscreen should work - Bug 1312334
+  - Fennec
+    - should play video when click on video element
+    - should adjust all the controls' size accordingly
```
Comment (timdream) [inline RayLin.md]: Have these bugs been filed?

File: RayLin.md (hunk 1 above)
Comment (raylin) [inline RayLin.md]: Sorry, it was landing then. Now I'm waiting for a few intermittent failed retry, if passed, I'll flag new `checked-needed`.

File: RayLin.md (hunk 2 above)
Comment (raylin) [inline RayLin.md]: I'll update this section soon later.

First regression is filed by Alastor which addresses similar issue.
I pointed out the second issue in one of the comment of Bug 1271765, and I'm gonna file this bug now.

**Pull request merged**